src/views/jobs/jobs.jsx

```diff
@@ -26,6 +26,8 @@ const Jobs = () => (
         <div className="bottom">
             <div className="inner">
                 <h3><FormattedMessage id="jobs.openings" /></h3>
+                We don&apos;t have any open positions at this time. Please check back soon for opportunities!
```

Comment:
Translation wanted

```suggestion
<FormattedMessage id="jobs.nojobs" />
```

and add the text to `l10n.json`